src/__tests__/http-test.ts

```diff
@@ -96,11 +92,11 @@ describe('GraphQL-HTTP tests for connect', () => {
 
     return {
       request: () => supertest(app),
-      use: (...args) => app.use(...args),
+      use: app.use.bind(app),
       // Connect only likes using app.use.
-      get: (...args) => app.use(...args),
-      put: (...args) => app.use(...args),
-      post: (...args) => app.use(...args),
+      get: app.use.bind(app),
+      put: app.use.bind(app),
+      post: app.use.bind(app),
```

Comment:
Can you please extract all stuff that changes code beyond typings into separate PR so I can merge it in Flow.
Also, the move of `write`.

Reply:
👍 Already extracted the `.write` changes earlier this morning: https://github.com/graphql/express-graphql/pull/641